neo/io/basefromrawio.py

```diff
@@ -71,7 +71,7 @@ def __init__(self, *args, **kargs):
         self.parse_header()
     
     def read_block(self, block_index=0, lazy=False, cascade=True, signal_group_mode=None, 
-                units_group_mode=None, load_waveforms=False, time_slices=None):
+                units_group_mode=None, load_waveforms=False, time_slices=None, channels_to_load='all'):
```

Comment:
This **channels_to_load** is that it refer the index inside the rawio.
For the general case when only one AnalogSignal with several channel at same sampling_rate this sounds goods.
But difficulties here for the user when it is not the genral case
Imagine that a IO give you for each segment 2 AnalogSignal:
  * the first shape (64, 10000) at 30kHz
   * the second is (20, 1000) at 1000Hz
Blackrock could and certainly will be have this if we read at the same time ns1 and ns5.
Internally the rawio have 84 channels so channels_to_load is in [0 ... 83].
I think this is confusing because if you when the channel 10 related to the second group of channel you specify channel_to_load=[74]